apiary.apib

```diff
@@ -39,6 +39,30 @@ Final version aligned with FIWARE Release 5.2.1.
 
 # Group aiakos v1
 
+## Regions [/v1/support]
+
+Get the region list with keys
+
+### Get list [GET]
+
```

Comment:
as you can see, I'm using  a 'JSON.stringify(regionsHashTable);' to generate JSON